download.html

```diff
@@ -11,11 +11,25 @@
 form label {
 	display: block;
 	padding: .2em;
+	padding-left: 1.9em;
+	page-break-inside: avoid;
+	break-inside: avoid;
+	-webkit-column-break-inside: avoid;
 }
 
-form input {
-	margin-right: .7em;
-}
+	form label > *:nth-child(2) {
```

Comment:
This selector is really brittle. Is there a better way of styling this?

Reply:
@mAAdhaTTah Fixed